### ISSUE TITLE
[MODULE] - NER-Deberta

### ISSUE THREAD

**Issue by Rashid-Ahmed** [code-kern-ai/bricks#307]
**Please describe the module you would like to add to bricks**
A Named Entity Recognition module hat uses a Deberta base model and is fine-tuned on few-NERD dataset.
The Token labels are Person, Organisation, Location, Building, Event, Product, Art & Misc.

**Do you already have an implementation?**
```python
from transformers import pipeline

def print_ner(sentences):
    """Cleaning and printing NER results

    """
    for sentence in sentences:
        last_entity_type = sentence[0]['entity']
        last_index = sentence[0]['index']
        word = sentence[0]['word']
        for i, token in enumerate(sentence):
            if (i > 0):
                if (token['entity'] == last_entity_type) and (token['index'] == last_index + 1):
                    word = word + '' + token['word']

                else:
                    word = word.replace('▁', ' ')
                    print(f"{word[1:]} {last_entity_type}")
                    word = token['word']
                last_entity_type = token['entity']
                last_index = token['index']

                if i == len(sentence) - 1:
                    word = word.replace('▁', ' ')
                    print(f"{word[1:]} {last_entity_type}")


pipe = pipeline(model='RashidNLP/NER-Deberta')
sentence = pipe(["Elon Musk will be at SpaceX's Starbase facility in Boca Chica for the orbital launch of starship next month"])
print_ner(sentence)
```

**Additional context**
https://huggingface.co/RashidNLP/NER-Deberta

**Comment by LeonardPuettmannKern** [comment]
Thanks @Rashid-Ahmed! Looks really cool. If it's on HuggingFace we can implement it very easy. 👍 Will look into it soon.

**Comment by LeonardPuettmannKern** [comment]
@Rashid-Ahmed implemented your model in PR #323 . My colleagues are going to merge it soon. Thank you for your contribution!

**Comment by LeonardPuettmannKern** [comment]
Added with #323
